1.10.5/CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+Astronomer Certified 1.10.5-7, 2020-04-27
+--------------------------------------------
+
+No changes in `astronomer/airflow` repo.
+
+Dockerfile changes are:
+
+- Upgrade sqlite3 packages for Alpine 3.10  ([commit](https://github.com/astronomer/ap-airflow/commit/2f29d493259cddd487bcc306b829a4ec4a74f35e))
+- Fix OpenSSL CVE ([commit](https://github.com/astronomer/ap-airflow/commit/6de11c2c87e78b7a3171d8fb222c7278fcb673c9))
```

Comment:
```suggestion
- Upgrade OpenSSL to mitigate CVE ([commit](https://github.com/astronomer/ap-airflow/commit/6de11c2c87e78b7a3171d8fb222c7278fcb673c9))
```

Plus we need to mention _which_ OpenSSL cve it is we fixed.

Reply:
Fixed